lib/routes.js

```diff
@@ -35,6 +35,29 @@ function checkUnsuportedRoutes(req, res, log) {
     return routesUtils.responseXMLBody(errors.MethodNotAllowed, null, res, log);
 }
 
+// current function utility is minimal, but will be expanded
+export function isHealthy() {
+    return true;
+}
+
+export function setHealthCheckResponse(req, res, log) {
+    if (isHealthy()) {
+        if (req.method === 'GET' || req.method === 'POST') {
+            res.writeHead(200, { 'Content-Type': 'application/json' });
+            res.write('{}');
+            log.debug('healthcheck response',
+                { httpMethod: req.method, httpCode: res.statusCode });
+        } else {
+            res.writeHead(400, { 'Content-Type': 'application/json' });
```

Comment:
@rahulreddy Added the extra line to master.json! And it looks like we're keeping the 400